Parser/tokenizer.c

```diff
@@ -1390,8 +1390,27 @@ tok_get(register struct tok_state *tok, char **p_start, char **p_end)
         case 'u':
         case 'U':
             c = tok_nextc(tok);
-            if (c == 'r' || c == 'R')
+            if (c == 'r' || c == 'R') {
+                if (Py_Py3kWarningFlag) {
+                    if (PyErr_WarnExplicit_WithFix(PyExc_Py3xWarning, "the 'ur' prefix in string literals is not supported in 3.x", 
```

Comment:
missing colon in this message at end of first line?

Reply:
The error formatter for this function adds the colon.

Reply:
This PR had some error using the other method where the colon was explicit, I will then fix that so that the colon is picked up from the message and warning formatter functions.